### DISCUSSION TITLE
Hello, I just made a pull request that adds a ton more numbers (up to 20800000000, I believe)

### DISCUSSION THREAD

**Discussion by wireboy5** [samuelmarina/is-even#75] [General]
This pull request (#73), however, uses lookup tables instead of if statements. This is to circumvent the github file size limit. If you feel that this pull request does not fit the spirit of the project, go ahead and close it.

Also, good luck reviewing the lookup tables lol. 2.6GB

Happy Coding!

**Comment by Shashank3736** [comment]
Automating things is a no-go type it with your own hands. 👎
